HW2.R

```diff
@@ -0,0 +1,76 @@
+#########
+###HW2###
+#########
+
+read.csv("iris.csv") #read in data set
+iris #to look at the dataset
+dim(iris) #to look at the dimensions.150 by 5.
+names (iris) #to look at all the names
+names(iris) <- c("sepalL", "sepalW",  "petalL", "petalW", "species") #rename
+names(iris) #to check to see if I did it right
+x <- iris$sepalL #rename what I plan on using for my regression
+x #check to make sure it works
+y <- iris$petalL #rename what I plan on using for my regression
+y #check
+fit <- lm(y ~x) #two things are happening here. One the lm function is showing me the intercept and slope of my regression ,and I renamed these coefficients fit
+fit #check
+b <- coef( fit ) # grabs the coefficients 
+b #checking my work
+int <- b[1] # grabbing and renaming JUST the intercept
+slope <- b[2] #grabbing and renaming JUST the slope
+plot(x, y) # plotting x and y, Sepal Length and Petal Length LEAVE THIS WINDOW OPEN
+abline(fit) #adding a regression line
+
+######################
+###regression error###
+######################
+
+yhat <- slope*x + int # the fitted values
+points( x, yhat, pch=3)  # adding the fitted points
+i=150 #made i =150
+segments( x[i], y[i], x[i], yhat[i], col="red" )
+segments( x, y, x, yhat, col="red", lty=2 ) #created lines and made them red and checkered
+error <- yhat - y #named error the error
+error #to check
+sse <- sum((error)^2) #named the sum of all errors squared sse
+text( 5, 7, paste("SSE:111.4592")) #added sum of all error squared
```

Comment:
I see you changed this in the wrapped function, otherwise the text won't change when you change x and y. I had issues with this haha.